### DISCUSSION TITLE
Error are coming when run UI5 application with wdi5

### DISCUSSION THREAD

**Discussion by pragu4** [ui5-community/wdi5#397] [Q&A] [answered]
Hello SAP Team,

I created a simple ui5 application in SAP BAS then performed the Below 2 node.js commands then errors are come up.

Please provide the solution so we will be working on testing the sapui5 application.

npm init wdi5
npm run wdi5

Error :- 

user: finaljs $ npm run wdi5

> finaljs@0.0.1 wdi5
> wdio run ./wdio.conf.js

Execution of 0 workers started at 2023-01-11T01:01:14.968Z

SevereServiceError in "onPrepare"
SevereServiceError: 
A service failed in the 'onPrepare' hook
SevereServiceError: Couldn't start Chromedriver: timeout
Chromedriver failed to start.
    at ChromeDriverLauncher.onPrepare (/home/user/projects/finaljs/node_modules/wdio-chromedriver-service/build/launcher.js:106:13)
    at async /home/user/projects/finaljs/node_modules/@wdio/cli/build/utils.js:38:17
    at async Promise.all (index 0)
    at async Launcher.run (/home/user/projects/finaljs/node_modules/@wdio/cli/build/launcher.js:86:13)

Stopping runner...
    at /home/user/projects/finaljs/node_modules/@wdio/cli/build/utils.js:54:35
    at async Launcher.run (/home/user/projects/finaljs/node_modules/@wdio/cli/build/launcher.js:86:13)
HookError [SevereServiceError]: 
A service failed in the 'onPrepare' hook
SevereServiceError: Couldn't start Chromedriver: timeout
Chromedriver failed to start.
    at ChromeDriverLauncher.onPrepare (/home/user/projects/finaljs/node_modules/wdio-chromedriver-service/build/launcher.js:106:13)
    at async /home/user/projects/finaljs/node_modules/@wdio/cli/build/utils.js:38:17
    at async Promise.all (index 0)
    at async Launcher.run (/home/user/projects/finaljs/node_modules/@wdio/cli/build/launcher.js:86:13)

Stopping runner...
    at /home/user/projects/finaljs/node_modules/@wdio/cli/build/utils.js:54:35
    at async Launcher.run (/home/user/projects/finaljs/node_modules/@wdio/cli/build/launcher.js:86:13) {
  origin: 'onPrepare'
}

Thanks
Pragnesh

**Comment by dominikfeininger** [comment, marked as answer]
Hi @pragu4,

can you please provide

Runtime Env :

wdi5/wdio-ui5-service-version: 
UI5 version: 
wdio-version (output of wdio --version): 
node-version (output of node --version): 
Browser + Version :

**Comment by Siolto** [comment]
@pragu4,

as already discussed in following [issue](https://github.com/ui5-community/wdi5/issues/329), BAS currently does not support wdi5. You can also vote on the [customer incident ticket](https://influence.sap.com/sap/ino/#/idea/287947) that was opened for that reason.

As long as BAS does not support wdi5 we recommend to use other IDEs like VSCode.

**Reply by vobu** [reply]
🤣

**Comment by vobu** [comment]
thanks for coining as a SAP Team, but we're a merry band of UI5ers only, doing Open Source development 😉  
and to add to the cause: the SAP BAS team was not yet able to add support for `wdi5` tests in BAS - see #329 and the corresponding influence request at https://influence.sap.com/sap/ino/#/idea/287947
I'll close this issue for now as you can follow above mentioned #329 for getting `wdi5` support in BAS. Hope this is ok for you!
